webpack.config.js

```diff
@@ -0,0 +1,86 @@
+const path = require('path');
+const webpack = require('webpack');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+
+module.exports = {
+  target: 'web',
+  entry: './src/index.js',
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'vendor.js',
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(css|scss)$/,
+        use: [MiniCssExtractPlugin.loader, 'css-loader', 'sass-loader'],
```

Comment:
Here I think you need and autoprefixer for css. Also I'm not seeing sourcemap configuration, both for js and css.

Reply:
If things are missing, this is the initial